### DISCUSSION TITLE
Standalone Assistant

### DISCUSSION THREAD

**Discussion by tziemek** [artiso-solutions/CoVoX#20] [Ideas]
Did you by any chance investigate https://mycroft.ai/ ? It's a voice assistant which can run onprem, i.e. a raspberry without any internet connection. Could be really nice for our customers who are still afraid to connect their devices to the cloud.

**Comment by tommasobertoni** [comment]
We discussed the option of going offline mainly to try to improve performance, since those of Cognitive Services are not as we'd like.
[DeepSpeech](https://github.com/mozilla/DeepSpeech) looked promising, but we decide to schedule the offline-implementation for a next release.

**Comment by vgibilmanno** [comment]
Looks very interesting I'll have a deeper look into this.
